src/content/certifications/exam-faqs/consul-associate-003.mdx

```diff
@@ -0,0 +1,117 @@
+<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
+
+## Exam Availability
+
+Coming soon!
+
+## Which Exam to Take
+
+The [Consul Associate 002 certification](https://developer.hashicorp.com/certifications/networking-automation#consul-associate-002-details) is still relevant and will be accepted as validation of Consul knowledge until each individual badge’s expiration date. Receiving a Consul Associate 003 certification will not impact any existing Consul Associate 002 credentials, and a single candidate can hold both at the same time. However, either certification alone can be used to validate Consul knowledge at the associate level.
```

Comment:
since this is on devdot we don't need to include the full path

```suggestion
You can use either exams to validate Consul knowledge at the associate level. The [Consul Associate 002 certification](/certifications/networking-automation#consul-associate-002-details) is still relevant and will be accepted as validation of Consul knowledge until the badge’s expiration date. Receiving a Consul Associate 003 certification will not impact any existing Consul Associate 002 credentials. You can hold both at the same time.
```